### ISSUE TITLE
Filter - Date should be displayed as Publication Date

### ISSUE THREAD

**Issue by wendylow1** [knowledgefutures/priorart-site#37]
Currently in the Filter, there are two options:  1)  Date and 2)  Relevancy.  'Date' should be displayed as 'Publication Date'.  This will be more clear as we have two dates, Publication Dates and Uploaded Dates, that are returned from search results.

**Comment by metasj** [comment]
Agreed, thanks Wendy.   We need to revisit Relevancy in general -- it is currently not helpful for search and takes a lot of space in the sidebar.  If we include it at all it should extend further back than 10 years.
